Backfill_Correction/delphiBackfillCorrection/R/preprocessing.R

```diff
@@ -0,0 +1,222 @@
+## Data Preprocessing
+## 
+## The raw input data should have 4/5 basic columns:
```

Comment:
for files with only 4 columns, which columns are optional?

Reply:
`issue date` and `lag`, we should have at least one of it, since the other one can be computed accordingly given `time_value`

Reply:
So, the 4 columns would be `time_value`, `lag`/`issue_date`, `denom`, `num`